doc/news/_preparation_next_release.md

```diff
@@ -118,6 +118,14 @@ you up to date with the multi-language support provided by Elektra.
 - Introduced `KeySet::remove(Key)` and `KeySet::remove(String)`
 - Removed `KeySet::lookup(Key, int)` and `KeySet::lookup(String, int)` as well as accompanying flag definitions `KeySet::KDB_O_NONE`, `KeySet::KDB_O_DEL` and `KeySet::KDB_O_POP`. Please use `KeySet::lookup(Key)` and `KeySet::lookup(String)` instead. Instead of `KeySet::KDB_O_DEL`, please consider using `Key::release`. The proper replacement for `KeySet::KDB_O_POP` is `KeySet::remove(Key)` or `KeySet::remove(String)`.
 - Native library proxy interface `Elektra` is now package private (previously was public).
+- Added example Java JNI plugin `whitelist`
+- Changed `Key nextMeta()` to `Optional<Key> nextMeta ()` no longer throwing NoSuchElementException for non-exceptional behavior
+- Added support of binary valued keys:
+  - Renamed `KeyBinaryTypeNotSupportedException` to `KeyStringValueException`
+  - Introduced `KeyBinaryValueException`
+  - Improved `Key` test coverage
+  - Introduced `Key::getBinary()` and `Key::setBinary(byte[])`
+- Fixed example project in `examples/external/java/read-keys-example`
```

Comment:
What was fixed? A typo?

Reply:
Before it was not working at all... I will add a few words about that

Reply:
done